### ISSUE TITLE
Merge with DuckDB v1.1.1

### ISSUE THREAD

**Issue by Dtenwolde** [cwida/duckpgq-extension#151]
### What happens?

DuckDB released a new version, v1.1.1 https://github.com/duckdb/duckdb/releases/tag/v1.1.1
We should update the extension to be available as a community extension again. 

For instructions: 
```bash
cd duckdb-pgq
git checkout main
git fetch --tags upstream
git merge <latest_version>
```
Fix any merge conflicts
```bash
git add <fixed merges>
```
Generate the parser
```bash
python scripts/generate_flex.py
python scripts/generate_grammar.py
```
Build debug & release
```bash
cd ..
make clean
make debug GEN=ninja
make GEN=ninja
./build/debug/unittest --test-dir ../../../ test/sql/*
```

### To Reproduce

-

### OS:

macOs 13 - Apple M1 Pro

### DuckDB Version:

v1.1.1

### DuckDB Client:

CLI

### Full Name:

Daniel ten Wolde

### Affiliation:

CWI

### How did you load the extension?

Community extension version

### Did you include all relevant data sets for reproducing the issue?

Yes

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by dgg32** [comment]
Hi. When will the new version be packaged and uploaded to the community channel, please? Currently, DBeaver cannot install from S3 (missing the file) and force install (version conflict). Thank you!

**Comment by Dtenwolde** [comment]
Hey there! I was on holiday for the last two weeks, so the extension is not yet available as a community extension. I'll try to get it working asap :)

**Comment by Dtenwolde** [comment]
https://github.com/duckdb/community-extensions/pull/135 
DuckPGQ is now available as a community extension again!
Closing this issue as we have now fully merged with DuckDB v1.1.1
